Source/JavaScriptCore/runtime/FunctionConstructor.cpp

```diff
@@ -66,23 +66,24 @@ void FunctionConstructor::finishCreation(VM& vm, FunctionPrototype* functionProt
     putDirectWithoutTransition(vm, vm.propertyNames->prototype, functionPrototype, PropertyAttribute::DontEnum | PropertyAttribute::DontDelete | PropertyAttribute::ReadOnly);
 }
 
-static String stringifyFunction(JSGlobalObject* globalObject, const ArgList& args, const Identifier& functionName, FunctionConstructionMode functionConstructionMode, ThrowScope& scope, std::optional<int>& functionConstructorParametersEndPosition)
+ASCIILiteral functionConstructorPrefix(FunctionConstructionMode functionConstructionMode)
 {
-    ASCIILiteral prefix;
     switch (functionConstructionMode) {
     case FunctionConstructionMode::Function:
-        prefix = "function "_s;
-        break;
+        return "function "_s;
     case FunctionConstructionMode::Generator:
-        prefix = "function* "_s;
-        break;
+        return "function* "_s;
     case FunctionConstructionMode::Async:
-        prefix = "async function "_s;
-        break;
+        return "async function "_s;
     case FunctionConstructionMode::AsyncGenerator:
-        prefix = "async function* "_s;
-        break;
+        return "async function* "_s;
     }
+    return ASCIILiteral { };
```

Comment:
nits: should add `ASSERT_NOT_REACHED();` ?

Reply:
Changed.